client/ayon_core/hosts/max/hooks/pre_copy_mxp.py

```diff
@@ -0,0 +1,23 @@
+from ayon_applications import PreLaunchHook, LaunchTypes
+from ayon_core.hosts.max.lib import create_workspace_mxp
+
+
+class PreCopyMxp(PreLaunchHook):
+    """Copy workspace.mxp to workdir.
+
+    Hook `GlobalHostDataHook` must be executed before this hook.
+    """
+    app_groups = {"3dsmax", "adsk_3dsmax"}
```

Comment:
Out of curiosity - why do both of these groups exist? :/

Reply:
i remember there is a bug for launching 3dsmax when there is no `adsk_3dsmax` in the `app_groups`. Guess we can create the github issue and discuss what to do to fix it as it is not related to this PR.